test/v2/retrieveAccountingType/model/response/RetrieveAccountingTypeResponseSpec.scala

```diff
@@ -38,20 +53,60 @@ class RetrieveAccountingTypeResponseSpec extends UnitSpec {
       |""".stripMargin
   )
 
+  private val invalidMultipleIncomeSourcesDownstreamResponse = Json.parse(
+    """
+      |{
+      |  "selfEmployments": [
+      |    {
+      |      "incomeSourceId": "AT0000000000001",
+      |      "incomeSourceName": "string",
+      |      "cessationDate": "2019-08-24",
+      |      "commencementDate": "2019-08-24",
+      |      "latency": {},
+      |      "accountingPeriodStartDate": "2019-08-24",
+      |      "accountingPeriodEndDate": "2019-08-24",
+      |      "accountingType": "CASH",
+      |      "quarterReporting": {},
+      |      "basisPeriodStartDate": "2019-08-24",
+      |      "basisPeriodEndDate": "2019-08-24",
+      |      "obligations": []
+      |    }
+      |  ],
+      |  "ukProperty": [
+      |    {
+      |      "incomeSourceType": "02",
+      |      "incomeSourceId": "AT0000000000001",
+      |      "incomeSourceName": "string",
+      |      "cessationDate": "2019-08-24",
+      |      "commencementDate": "2019-08-24",
+      |      "latency": {},
+      |      "accountingPeriodStartDate": "2019-08-24",
+      |      "accountingPeriodEndDate": "2019-08-24",
+      |      "accountingType": "CASH",
+      |      "quarterReporting": {},
+      |      "basisPeriodStartDate": "2019-08-24",
+      |      "basisPeriodEndDate": "2019-08-24",
+      |      "obligations": []
+      |    }
+      |  ]
+      |}
+      |""".stripMargin
+  )
+
   private val parsedRequestBody = RetrieveAccountingTypeResponse(AccountingType.CASH)
 
   "RetrieveAccountingType" should {
     "parse downstream response correctly given valid Json" in {
-      validDownstreamResponse.as[RetrieveAccountingTypeResponse] shouldBe parsedRequestBody
+      validSelfEmploymentsDownstreamResponse.as[RetrieveAccountingTypeResponse] shouldBe parsedRequestBody
     }
   }
 
-  "write to Json correctly" in {
-    Json.toJson(parsedRequestBody) shouldBe validDownstreamResponse
-  }
-
   "throw an error given invalid Json" in {
     assertThrows[JsResultException](invalidDownstreamResponse.as[RetrieveAccountingTypeResponse])
   }
 
+  "throw an error when multiple income sources are returned" in {
```

Comment:
This is not a valid test scenario as businessID is supplied as a path param so multiple income sources will never be present. It will be one of the three depending on the supplied ID.